docs/plugin_developers_guide/09_packaging.rst

```diff
@@ -0,0 +1,256 @@
+.. Documentation of package for a Plugin
+
+Packaging Documentation
+=======================
+
```

Comment:
_Adding an introduction that talks about the fact every plugin is in its own repository would be worthwhile._